README.md

```diff
@@ -30,16 +30,18 @@ You'll also need to clone a new `.env` file from the `.env.tempalate` to store l
 $ cp .env.template .env  # (first time only)
 ```
 
-The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [TRELLO_KEY],[TRELLO_TOKEN] and [TRELLO_BOARD] variables which are used to set the Trello board and permissions.
```

Comment:
Nice spot updating this! We also need to add some notes for how to run the tests that you've added, including any prerequisites (like firefox/geckodriver)

Reply:
yes, added these in now.